data/implementations/features/api-features.yaml

```
@@ -1,8 +1,8 @@
 category_id: api-features
 features:
-  - id: api-external-column-data
-    display_name: External column data
-    note: "In parquet.thrift: ColumnChunk->file_path"
+  - id: api-parquet-summary-file
+    display_name: Parquet summary file
+    note: "Files named _metadata that consolidate footers from multiple parquet files."
```

I think the reference to the `file_path` field will help keep this specific. So perhaps something like 

```suggestion
    note: "Files named _metadata that consolidate footers from multiple parquet files, stored in ColumnChunk->file_path"
```

---

Is "parquet summary file" really the same as supporting `file_path`?

Maybe summary files are the only known use case of actually using `file_path`? But nothing in the spec says file_path is necessarily associated with "summary files". In fact is there anything in the spec about summary file or is that just an implementation choice some people made?

I'm not opposed to having a row for support of summary files, but that seems like a different question than `file_path` support?